src/parser/detectHandler.test.js

```diff
@@ -6,6 +6,10 @@ describe('detectHandler', () => {
     expect(detectHandler('!ac signup')).toEqual(handlers.get('signup'))
   })
 
+  test('it should properly parse xp command', () => {
+    expect(detectHandler('!ac xp username')).toEqual(handlers.get('xp'))
```

Comment:
Can we make the command just "!xp username" instead? !ac isnt relevant IMO

Reply:
Command changed to !xp 👍